### PR TITLE
TTT cleverer in progress

### PR DESCRIPTION
**Pull request by ghost**
### Summary

_What this work does, and why it's needed._

cleverer TTT computer player + game. Still working on it.

### What kind of change does this PR introduce?

- [ ] Bug fix
- [ ] Add a new feature/page

Related issue: 

### How to verify locally

1. `npm start`
1. http://localhost:8080/
 

**Expected results**:

### CODE REVIEW

File: _includes/layouts/kids.njk [hunk 1]
```diff
@@ -5,71 +5,123 @@ templateClass: tmpl-kids
...
+            // middle col win
+                if ((board.charAt(1) !== '-') && (board.charAt(1) === board.charAt(4)) && (board.charAt(4) === board.charAt(7)) {}
+            // right col win
+                if ((board.charAt(2) !== '-') && (board.charAt(2) === board.charAt(5)) && (board.charAt(5) === board.charAt(8)) {}
         }
-        function play() {
-            // play Tic Tac Toe
-            console.log("board.one", board.one)
-            console.log("play TTT")
+        const play =() => {
```
Comment (ghost) [inline _includes/layouts/kids.njk]: @twalker I'm interested in a particular kind of question I don't have the answer to. I was curious what you think might make sense. So, imagine a team of bright people, but not that experienced with tech stuff. In fact, maybe one of them won a nobel prize in poetry, and can jam with the best of em! But we still have to make decisions about workflow, and about code repo patterns. Here is why I'm interested in this. With software engineers, they tend to be more comfortable with switching processes, switching from one technology to another, switching from one way of doing things to another. But, that trait of switching stuff, is really a software engineering thing, it's not a bright person thing. And this means we have to be very careful and intentional about how we design our processes, workflows, and code patterns as a technical writing team.

Comment (ghost) [reply _includes/layouts/kids.njk]: So, with your experience using Eleventy so far, do you have some thoughts to share about what to pay attention to when starting a new project?

Comment (twalker) [reply _includes/layouts/kids.njk]: In hindsight, I chose too complex of a starter project in Eleventy. I don't regret choosing Eleventy over Jekyll, but should have started with more of a bare bones Eleventy starter with minimal configuration. I thought having a more fully functional starter project would be the most flexible, but suspect it made the project less approachable to contribute to. 

If I had a chance to restart, I'd started us with a more basic setup, knowing the more advanced functionality, such as tags, multiple layouts, etc., would be available if we needed it. The team could grow into more advanced features organically, without all the bells and whistles already in place.

I should have considered my primary audience more than satisfying my own curiosity. So, the lesson to me is to keep the readers at the center of decisions. The same spirit echos in the "Keep It Stupid Simple" (KISS) principle, as well as the "You Ain't Gonna Need It" (YAGNI) principle.

**Comment by ghost** [comment]
Made a bit more progress. I'll return to this in another PR before friday.

**Pull request merged**